contributed_definitions/nyaml/NXsample_history.yaml

```diff
@@ -0,0 +1,17 @@
+category: base
+doc: |
+  A set of physical processes that occurred to  the sample prior/during experiment.

```

Comment:
"A possilibility to link in a reference to another set of pieces of information where one can learn about physical processes or other activities which were performed **before** ..." (the instance where this NXsample_history base class is hooked into)

Reply:
Not sure that "before" is correct here. We could also keep track of physical processes that are occurring **during** the measurement, e.g. in transient experiments. However, this may be quite tricky, so "before" may be better.